ui_framework/src/components/form/form_row/form_row.js

```diff
@@ -116,7 +116,7 @@ KuiFormRow.propTypes = {
   id: PropTypes.string,
   isInvalid: PropTypes.bool,
   error: PropTypes.oneOfType([PropTypes.string, PropTypes.arrayOf(PropTypes.string)]),
-  helpText: PropTypes.string,
+  helpText: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
```

Comment:
Just curious -- what's this change for?

Reply:
Sorry, just read the PR description more closely. I think we should change this to:

```javascript
helpText: PropTypes.node,
```

This will accept strings, elements, and arrays, which is what I think you're for. Can we also update the doc site with an example of this in action?